spec/spec_helper.rb

```diff
@@ -27,6 +27,7 @@
   add_filter 'lib/traject/tei_config.rb'
```

Comment:
@cbeer I tried that. When doing that, it disregards all the filters in the block.